docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -224,7 +230,14 @@ To filter the events returned from your queries, use the following data structur
 
 ## Filtering events for subscription
 
-To create a subscription, you can set a filter to return only the set of events you're interested in listening for. Unlike filtering event queries, you are able to combine subscription filters.
+To create a subscription, you can set a filter to return only the set of events you're interested in listening for. 
+
+:::caution
+
+This set of filters only applies to **Event subscription APIs**. It differs from the filters offered for the query API (see previous section), and in particular it **does** support combinations, like `"All": [...]`, `"Any": [...]`, `"And": [_, _]`, `"Or": [_, _]`, or `"Not": _`. 
```

Comment:
```suggestion
This set of filters applies only to event subscription APIs. It differs from the filters offered for the query API (see previous section). In particular, it supports combinations like `"All": [...]`, `"Any": [...]`, `"And": [_, _]`, `"Or": [_, _]`, and `"Not": _`. 
```
Again, nit pile. Just in case you use the prev suggestion so it's matchy matchy.